Dockerfile

```diff
@@ -16,8 +18,33 @@ RUN curl -L https://github.com/facebook/mcrouter/archive/${MCROUTER_SHA}.tar.gz
 
 # Build folly, a dependency of mcrouter
 WORKDIR /tmp/folly-${FOLLY_SHA}/folly
-RUN apt-get -y update && ./build/deps_ubuntu_12.04.sh
-RUN apt-get -y install libboost-program-options1.54-dev
+RUN apt-get -y update && if [ ${RELEASE} = "12.04" ]; then \
+	./build/deps_ubuntu_12.04.sh && apt-get -y install libboost-program-options1.54-dev; else \
+	apt-get install -y \
```

Comment:
I wonder if we could get upstream to add a `./build/deps_ubuntu_14.04.sh` similar to the 12.04 script.  That'd make building this much easier and would allow the project to own their dependency list.